members/admin/union_admin.py

```diff
@@ -28,15 +31,20 @@ def get_queryset(self, request):
             "Bestyrelsen",
             {
                 "fields": (
+                    "chairman_old",
+                    "chairman_email_old",
+                    "second_chair_old",
+                    "second_chair_email_old",
+                    "cashier_old",
+                    "cashier_email_old",
+                    "secretary_old",
+                    "secretary_email_old",
+                    "board_members_old",
                     "chairman",
-                    "chairman_email",
                     "second_chair",
-                    "second_chair_email",
                     "cashier",
-                    "cashier_email",
                     "secretary",
-                    "secratary_email",
-                    "boardMembers",
+                    "board_members",
```

Comment:
Det slog mig lige at vi nok skal dele dem her op. 
Hvis du i stedet  for det her
```python
        (
            "Bestyrelsen",
            {
                "fields": (
                    "chairman_old",
                    "chairman_email_old",
                    "second_chair_old",
                    "second_chair_email_old",
                    "cashier_old",
                    "cashier_email_old",
                    "secretary_old",
                    "secretary_email_old",
                    "board_members_old",
                    "chairman",
                    "second_chair",
                    "cashier",
                    "secretary",
                    "board_members",
                )
            },
        ),
```
Laver den som: 
```python
        (
            "Bestyrelsen nye felter",
            {
                "fields": (
                    "chairman",
                    "second_chair",
                    "cashier",
                    "secretary",
                    "board_members",
                )
            },
        ),
        (
            "Bestyrelsen gamle felter",
            {
                "fields": (
                    "chairman_old",
                    "chairman_email_old",
                    "second_chair_old",
                    "second_chair_email_old",
                    "cashier_old",
                    "cashier_email_old",
                    "secretary_old",
                    "secretary_email_old",
                    "board_members_old",
                )
            },
        ),
```
Gør du det nok nemmere at flytte rundt på dem senere.

Reply:
God ide, done